chromium_src/components/translate/content/renderer/isolated_world_util.cc

```diff
@@ -15,7 +15,8 @@ void AdjustedSetIsolatedWorldInfo(int32_t world_id,
   // Limit all network requests to the security origin.
   if (!translate::UseGoogleTranslateEndpoint()) {
     new_info.content_security_policy =
-        "default-src 'self' 'unsafe-eval' 'unsafe-inline'";
+        "default-src 'self' 'unsafe-eval' 'unsafe-inline';"
```

Comment:
I'd suggest using `default-src 'none'` and then adding the various bits that we need like `script-src 'self'` and `style-src 'self'` most likely. But starting with `default-src 'none'` does limit the capabilities of this component to what is needed.

Reply:
@fmarier 
Changed, please take a look.